python_ta/reporters/node_printers.py

```diff
@@ -690,6 +642,44 @@ def render_static_type_checker_errors(msg, _node=None, source_lines=None):
     "dict-item-type-mismatch": render_static_type_checker_errors,
 }
 
+ERROR_TO_ERROR_FUNCTION_MAPPING = {
```

Comment:
Name this `PEP8_ERROR_RENDERERS`. There's a lot to unpack with naming here:

- "pep8" specifies the type of error being described
- "renderer" is more accurate than "error function"
- "mapping" is okay (and indeed a reasonable naming convention for dictionaries) but not common in the PythonTA codebase; instead, naming the variable "RENDERERS" puts the emphasis on what the values in the dict are.